sunpy/map/sources/asos.py

```diff
@@ -0,0 +1,178 @@
+'''ASO-S Map subclass definitions, currently for the HXI payload is available.'''
+
+import astropy.units as u
+#from sunpy import log
+from sunpy.time import parse_time
+from sunpy.map.mapbase import GenericMap, SpatialPair
+
+__author__="FuYu@PMO, from ASO-S/HXI Team"
+
+__all__ = ['HXIMap']
+
+
+class HXIMap(GenericMap):
+    """
+    HXI Image Maps (single map or multi-maps from one .fits file).
+    
+    The Hard X-ray Imager (HXI, Zhang et al. 2019b; Su et al. 2019) is one of the three payloads 
+    of the advanced space-based solar observatory (ASO-S, Gan et al. 2019, 2023).
+    
+    The energy ranges are ~15-300 keV for imaging and ~10-300 keV for spectra (Su et al. 2024).
+
+    ASO-S was launched on 9 October 2022 (UTC+8).
+
+    References
+    ----------
+    * `ASO-S Mission Page <http://aso-s.pmo.ac.cn>`_
+    * `ASO-S Data Center <http://aso-s.pmo.ac.cn/sodc/dataArchive.jsp>`_
+    * `ASO-S/HXI Flare List <http://aso-s.pmo.ac.cn/hxi_flare/hxi_flare_list.html>`_
+    * `The Tests and Calibrations of the Hard X-ray Imager Aboard ASO-S <https://link.springer.com/article/10.1007/s11207-024-02392-x>`_
+    
+    History:
+    --------
+        2025 08 27, FuYu@PMO, ASO/HXI Team. rsun_meters use the default value from sunpy (not fix).
+             11 11, add 'fix_hxi_observer' keyword.
+    """
+
+    def __init__(self, data, header, **kwargs):
+        super().__init__(data, header, **kwargs)
+        asos_hxi, hxi = self.meta.get('ORIGIN').split(' ')[0], self.meta.get('INSTRUME','') #deal with different headers
```

Comment:
```suggestion
        asos_hxi, hxi = self.meta.get('ORIGIN').split(' ')[0], self.meta.get('INSTRUME','')
```
What are the different headers?